mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -53,7 +53,8 @@ private[feature] trait PCAParams extends Params with HasInputCol with HasOutputC
 
 /**
  * :: Experimental ::
- * PCA trains a model to project vectors to a low-dimensional space using PCA.
+ * PCA trains a model to project vectors to a lower dimensional space of the top [[PCA!.k]]
```

Comment:
Yah I took a look - it seems to generate the correct link and we use it elsewhere but it doesn't seem to really do much.